Cargo.toml

```diff
@@ -26,8 +26,9 @@ derive-new = "0.5"
 rmpv = "0.4.4"
 rust-embed = { version = "5.2.0", features = ["debug-embed"] }
 image = "0.22.3"
-nvim-rs = { git = "https://github.com/kethku/nvim-rs", features = ["use_tokio"] }
-tokio = { version = "0.2.9", features = ["blocking", "process", "time", "tcp"] }
+nvim-rs = { git = "https://github.com/KillTheMule/nvim-rs", branch = "master", features = ["use_tokio"] }
+tokio = { version = "1.1.1", features = ["full"] }
+tokio-util = "*"

```

Comment:
You'll probably want to put in a real semver number here, to prevent unexpected breakage :)